src/utils/components/PendingChanges/utils/helpers.ts

```diff
@@ -290,7 +293,7 @@ export const getChangedDedicatedResources = (
   vmi: V1VirtualMachineInstance,
   currentSelection: boolean,
 ): boolean => {
-  if (isEmpty(vm) || isEmpty(vmi)) {
+  if (isEmpty(vm) || isEmpty(vmi) || isInstanceTypeVM(vm)) {
```

Comment:
`isInstanceTypeVM` needs to change as this function test if vm its connected to instancetype or preference. ANd we experiment that [some customers use vms with preference but no instancetype](https://issues.redhat.com//browse/CNV-69441). So here we should test only if vm is connected to an instance type. I'll update my pr to change this behaviour. We assumed that those two things comes together every time . but its not always the case

Reply:
@upalatucci Are you good with this PR now? I don't want to lgtm it if you have any remaining concerns.